### DISCUSSION TITLE
Tag manager not showing tags after server update to 3.2.1

### DISCUSSION THREAD

**Discussion by LINCCLIBS** [shlinkio/shlink#1530] [Help wanted] [answered]
I just updated our Shlink server to version 3.2.1.  Everything seemed to go fine, but now one of my API keys with author only permissions can no longer see the tags they created when looking at "Manage tags".  Only one tag displays.

<img width="379" alt="shlink-3 2 1-OC-tag-manager" src="https://user-images.githubusercontent.com/9955397/187763317-d1f9782b-7b18-404b-a943-305bafd9f9c9.png">

When looking at "List short URLs", the tags created with this API key do display next to the URL.
<img width="630" alt="shlink-3 2 1-OC-list-short-urls" src="https://user-images.githubusercontent.com/9955397/187763847-19f1459f-3733-4645-89ea-31fd83c7e412.png">

My non-restricted API key, and new admin API keys created show all tags correctly.

I have restarted the server and allowed Shlink to rebuild /data/cache/app_config.php.  Any help in getting the tag manager working again for this API key would be greatly appreciated.

**Comment by acelaya** [comment]
This looks like a bug. Can you [open an issue](https://github.com/shlinkio/shlink/issues/new?assignees=&labels=bug&template=Bug.md) so that I don't forget to investigate?

**Reply by LINCCLIBS** [reply, marked as answer]
Thanks, [issue 1537](https://github.com/shlinkio/shlink/issues/1537) has been created.
